app/backend/src/tests/test_email.py

```diff
@@ -85,20 +164,21 @@ def test_signup_email(db):
     with session_scope(db) as session:
         token, expiry_text = new_signup_token(session, request_email)
 
-        def mock_send_smtp_email(sender_name, sender_email, recipient, subject, plain, html):
-            assert recipient == request_email
-            assert token.token in plain
-            assert token.token in html
-            return message_id
+    def mock_send_email(sender_name, sender_email, recipient, subject, plain, html):
+        assert recipient == request_email
+        assert token.token in plain
+        assert token.token in html
+        return message_id
 
-        with patch("couchers.email.send_smtp_email", mock_send_smtp_email):
-            send_signup_email(request_email, token, expiry_text)
+    with patch("couchers.email.send_email", mock_send_email):
+        send_signup_email(request_email, token, expiry_text)
```

Comment:
De-indenting these breaks the tests as then the `token` object doesn't have a DB session and sqlalchemy destroys it.